src/app/services/admin-patient.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { ServerApiService } from './server-api.service';
+import { Observable } from 'rxjs';
+import { Response } from '../../../api-objects/GenericResponse';
+import { AdminPatientResponse, ResetPasswordResponse, UserListItem } from '../../../api-objects/AdminPatient';
+import { PatientResponse } from '../../../api-objects/patientResponse';
+
+@Injectable()
+export class AdminPatientService {
+  constructor(private serverApiService: ServerApiService) { }
+
+  public createPatient(patientId : string, deceased : boolean, patientNotes : string) : Observable<Response<PatientResponse>> {
```

Comment:
I think it would be useful if you made an object to hold these variables and then add it to the api-objects folder: Refer to `ApiUsersCrudService.createUser(usersObj: UsersObject)` for a good reference of what I mean.

Reply:
createPatient could just take a PatientResponse, so when we add first name/last name/email we only have to update one object and not have two that are duplicates of each other.
Ironically the response only has to be a boolean success or failure to create a patient.

Reply:
We need to fix that so that the createPatient API returns the ID of the created Patient